app/ts/components/pages/ChangeChain.tsx

```diff
@@ -87,31 +87,27 @@ export function ChangeChain() {
 					</article>
 					<div class = 'media-content' style = 'padding-bottom: 10px'>
 						<div class = 'content'>
-							<p className = 'title' style = 'white-space: normal; text-align: center;'>
+							<p className = 'title' style = 'white-space: normal; text-align: center; padding: 10px;'>
 								<p className = 'title' style = 'white-space: normal; text-align: center; font-weight: bold;'>
 									{ chainChangeData.website.websiteOrigin }
 								</p>
 								would like to switch to
 								<p className = 'title' style = 'white-space: normal; text-align: center; font-weight: bold;'>
 									{ chainChangeData.chainName }
 								</p>
-								{ !chainChangeData.isInterceptorSupport && chainChangeData.simulationMode ?
-									<div style = 'font-size: 0.5em;'>
-										<ErrorContainer
-											text = { 'This chain is not supported by The Interceptor. If you want to use this chain anyway. Select Signing mode instead of Simulation mode and attempt to change the chain again. You will then be able to disable The Interceptor and send transactions without protection of The Interceptor.' }
-										/>
-									</div>
-								: <></> }
-								{ !chainChangeData.isInterceptorSupport && !chainChangeData.simulationMode ?
-									<div style = 'font-size: 1em;'>
-										<ErrorCheckBox
-											text = { 'This chain is not supported by The Interceptor. Would you like to disable The Interceptor and attempt to connect anyway?' }
-											checked = { connectAnyway }
-											onInput = { setConnectAnyway }
-										/>
-									</div>
-								: <></> }
 							</p>
+							{ !chainChangeData.isInterceptorSupport && chainChangeData.simulationMode ?
+								<ErrorContainer
+									text = { 'This chain is not supported by The Interceptor. If you want to use this chain anyway. Select Signing mode instead of Simulation mode and attempt to change the chain again. You will then be able to disable The Interceptor and send transactions without protection of The Interceptor.' }
```

Comment:
```suggestion
									text = { 'This chain is not supported by The Interceptor. If you want to use this chain anyway. Select Signing mode instead of Simulation mode and attempt to change the chain again. You will then be able to disable The Interceptor and send transactions without its protection.' }
```